runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java

```diff
@@ -103,6 +103,15 @@ public class EmoteClue extends ClueScroll implements LocationClueScroll
 			.orElseGet(Stream::empty)
 			.map(ItemRequirements::item)
 			.toArray(SingleItemRequirement[]::new));
+	private static final AnyRequirementCollection ANY_RUNE_HERALDIC_SHIELD = any("Rune shield (h)",
+		ItemVariationMapping.getVariations(RUNE_SHIELD_H1).stream()
+			.map(ItemRequirements::item)
+			.toArray(SingleItemRequirement[]::new));
+	private static final AnyRequirementCollection ANY_RUNE_HERALDIC_HELM = any("Rune helm (h)",
+		ItemVariationMapping.getVariations(RUNE_FULL_HELM).stream()
+			.filter(itemId -> itemId != RUNE_FULL_HELM && itemId != RUNE_FULL_HELM_G && itemId != RUNE_FULL_HELM_T)
```

Comment:
In my opinion this is a bug in our variation mapping, and should be fixed at the source (ie. https://github.com/abextm/runelite-cache-code-updater/issues/42) instead of this clunky workaround.